explanation/security.md

```diff
@@ -2,10 +2,18 @@ Anbox Cloud is secure by design, which means that its architecture, its componen
 
 Anbox Cloud uses the secure [LXD](https://ubuntu.com/lxd) for container and virtual machine management. To ensure security and isolation of each Android system, Anbox Cloud runs a single Android system per LXD instance.
 
-This security guide gives more insight into how security is ensured through different aspects of Anbox Cloud.
+Consider the following simple yet impactful measures to ensure that you run a secure Anbox Cloud deployment:
+
+- Always run the latest and supported version of Anbox Cloud. See [roadmap](https://discourse.ubuntu.com/t/release-roadmap/19359).
+- Do not set the `application.auto_update`, `instance.security_updates`, `container.security_updates` to `false`. See [AMS configuration](https://discourse.ubuntu.com/t/ams-configuration/20872).
+- Monitor resources used by applications regularly.
```

Comment:
I don't think we have existing docs on this. How do you typically monitor applications?

Reply:
We don't have something explicitly yet but we always wanted to create performance profiles to allow users identifying issues.

Reply:
Should we reword `applications` to `instances` ?